src/index.js

```diff
@@ -511,32 +518,17 @@ const initialize = async () => {
       }
     };
 
-    encryptMessageInput.onkeyup = () => {
-      if (
-        !getEncryptionKeyButton.disabled &&
-        encryptMessageInput.value.length > 0
-      ) {
-        if (encryptButton.disabled) {
-          encryptButton.disabled = false;
```

Comment:
Was this deleted to allow encrypting an empty message? I'm not sure why that was disallowed to being with :thinking: Does that work?